src/app/auth/profiles/profiles.component.ts

```diff
@@ -90,13 +90,22 @@ export class ProfilesComponent implements OnInit {
     reader.readAsText(file);
   }
 
+  /**
+   * Gets the required profiles.json file from the external repository
+   */
+   private fetchProfilesJson(): Promise<any> {
+    return fetch(AppConfig.clientDataUrl).then(res => res.json());
```

Comment:
```suggestion
   private fetchProfilesJson(): Promise<Profile[]> {
    return fetch(AppConfig.clientDataUrl).then(res => res.json().profiles || []);
```
I think can include some error handling here and specify the promise type

Reply:
Sounds good to me! :+1: